CI/flatpak/com.obsproject.Studio.json

```diff
@@ -44,6 +44,61 @@
     "*.la"
   ],
   "modules": [
+    {
+      "name": "plog",
+      "buildsystem": "cmake-ninja",
+      "config-opts": [
+        "-DPLOG_BUILD_SAMPLES=OFF"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/SergiusTheBest/plog.git",
+          "tag": "1.1.9",
+          "commit": "f47149410a4c927643148b96799f28b2d80d451b"
+        }
+      ]
+    },
+    {
+      "name": "usrsctp",
+      "buildsystem": "cmake-ninja",
+      "config-opts": [
+        "-Dsctp_build_shared_lib=OFF",
+        "-Dsctp_build_programs=OFF",
+        "-Dsctp_inet=OFF",
+        "-Dsctp_inet6=OFF",
+        "-Dsctp_werror=OFF",
+        "-DCMAKE_POSITION_INDEPENDENT_CODE=ON"
```

Comment:
I think you should explain why those flags are set.

Reply:
Added comment on why IP is turned off

Reply:
@tytan652 Is this comment enough explanation why these flags are set?

Reply:
I searched the "comment" in this conversation or in the commit description multiple times.
And I only find it recently in the manifest, please don't do that.
Do it in the commit description, please.

And you only explain two of the four that I selected.

Reply:
@tytan652 I have updated the commit message. Can you please resolve this conversation now. thanks